common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomResourceStore.java

```diff
@@ -374,32 +403,31 @@ private void fixCustomResourceDefinition(final JsonObject customResourceJsonElem
 
 					this.fixOrgId(formDefinition, "tenant");
 
-					changeProjectIdBetweenOrganizations(formDefinition);
+					this.changeProjectIdBetweenOrganizations(formDefinition);
 				}
 			}
 		});
 	}
+
 	/**
-	 * Fixes the organization id / tenant id in the given object with the one set in the configuration
+	 * Fixes the organization id / tenant id in the given object with the one set in the configuration.
+	 * @param jsonObject
+	 * @param key
 	 */
-	private void fixOrgId(final JsonObject jsonObject, String key) {
+	private void fixOrgId(final JsonObject jsonObject, final String key) {
 		if (jsonObject.has(key)) {
 			jsonObject.remove(key);
 		}
 
 		jsonObject.addProperty(key, this.currentOrganizationId);
 	}
 
+	/**
+	 * Fixes the project id in the given object with the one set in the configuration.
+	 * @param customResourceJsonElement
+	 */
 	private void changeProjectIdBetweenOrganizations(final JsonObject customResourceJsonElement) {
-		String defaultProjectId = this.restClient.getProjectId();
-		if (defaultProjectId != null) {
-			if (customResourceJsonElement.get("projectId") != null
-				&& !customResourceJsonElement.get("projectId").getAsString().equals("")) {
-				customResourceJsonElement.remove("projectId");
-				customResourceJsonElement.add("projectId", new JsonPrimitive(defaultProjectId));
-			}
-		} else
-			customResourceJsonElement.remove("projectId");
+		VraNgProjectUtil.changeProjectIdBetweenOrganizations(this.restClient, customResourceJsonElement);
```

Comment:
Why don't we call `VraNgProjectUtil.changeProjectIdBetweenOrganizations` directly?